cloudigrade/api/tests/util/test_get_max_concurrent_usage.py

```diff
@@ -69,10 +44,10 @@ def test_single_rhel_run_result(self, mock_schedule_concurrent_calc):
     @util_helper.clouditardis(util_helper.utc_dt(2019, 5, 3, 0, 0, 0))
     def test_single_rhel_run_no_result(self):
         """
-        Test with a single RHEL instance requesting results in futures.
+        Test with a single RHEL instance requesting results in the future.
 
-        Also assert that no ConcurrentUsage is saved to the database because the
-        requested date is in the future, and we cannot know that data yet.
+        We expect None because the we refuse to calculate data for future dates.
```

Comment:
The we?

Reply:
Whoops! Good catch. :) I just amended the specific commit that introduced that change and force-pushed everything back up.